src/Symfony/Component/Yaml/Inline.php

```diff
@@ -104,6 +108,18 @@ public static function dump($value, $exceptionOnInvalidType = false, $objectSupp
 
                 return 'null';
             case is_object($value):
+                if ($value instanceof \DateTime || $value instanceof \DateTimeImmutable) {
+                    if ($value->getTimezone()->getName() === date_default_timezone_get()) {
```

Comment:
I'm not sure about this. The code parsing the date might have a different default timezone. It is better to always include all info IMO